ui/components/multichain/nft-item/nft-item.tsx

```diff
@@ -27,6 +27,10 @@ import {
   getTestNetworkBackgroundColor,
 } from '../../../selectors';
 import { NFT } from '../asset-picker-amount/asset-picker-modal/types';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { ENVIRONMENT_TYPE_FULLSCREEN } from '../../../../shared/constants/app';
+// eslint-disable-next-line import/no-restricted-paths
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
```

Comment:
Hmm this rule is meant to prevent the UI from requesting stuff from the background/scripts.
Is there another `getEnvironmentType` or utility we can use here?

Reply:
As far as I know, this is the only utility. We're using it in a few places already. Do you know of another one we can use instead?

Reply:
Aw damn. No I don't know of another one we can use.

Thats okay then 😄

Reply:
Looking at the implementation, I'm not sure why this can only be used in the background.
https://github.com/MetaMask/metamask-extension/blob/main/app/scripts/lib/util.ts#L55-L56

I think it can be moved to the shared folder in the future.